recipes/czifile/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "czifile" %}
+{% set version = "2019.7.2" %}
+{% set sha256 = "04c0e6bed3b24d1bf42bc2cf899a5a08986641379305ce88600fd1c710486436" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  noarch: python
+  entry_points:
+    - czifile = czifile.czifile:main
+    - czi2tif = czifile.czi2tif:main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
Looking at the `setup.py`, there are some `install_requirements`:

```
 'numpy>=1.11.3',
 'tifffile>=2019.7.2',
 'imagecodecs>=2019.5.22;platform_system=="Windows"',
```

I'm not sure how important the version restrictions are. Also, what about `imagecodecs-lite` vs `imagecodecs` with a platform dependency?

Reply:
I've added version requirements. For `imagecodecs-lite`, the first release after `2019.5.22` (no corresponding `-lite` release exists).

The whole `imagecodecs` vs. `imagecodecs-lite` is a larger issue … `imagecodecs` is not yet a `conda-forge` package (see https://github.com/conda-forge/staged-recipes/pull/6842 for the lengthy discussion). Since much of `czifile`'s functionality is available with just `imagecodecs-lite`, I'd pick that as of the time being; I can change the dependency later when an `imagecodecs` package emerges.

(Concerning the windows specifier, I assume cgohlke pointed to that package, since he builds binary wheels for Windows, and hence there an installation is rather easy.)